lighthouse/classes/services/cherrytrack.py

```diff
@@ -35,3 +36,11 @@ def get_samples_from_source_plates(self, source_barcode):
 
     def filter_pickable_samples(self, sample):
         return sample["picked"]
+
+    def get_wells_from_destination_plate(self, destination_barcode):
```

Comment:
Is this just acting as an alias?

Reply:
It also makes sure it raises the exception if response is not right

Reply:
OK, is there a reason the other method does not raise an exception instead?

Reply:
The associated call under it may be successful but the response not be right. In that case we would need to raise an error to be consistent.